pyproject.toml

```diff
@@ -73,6 +73,7 @@ before-test = [
   # Ref: https://github.com/pypa/cibuildwheel/issues/1666
   "PIP_CONSTRAINT= pip install PyYAML",
 ]
+free-threaded-support = true
```

Comment:
This got replaced with `
enable = ["cpython-freethreading"]` which https://iscinumpy.dev/post/cibuildwheel-3-0-0/ also emphasizes.